lib/interface/screens/character_sheet_screen.dart

```diff
@@ -140,7 +139,7 @@ class _CharacterSheetScreenState extends State<CharacterSheetScreen> {
             children: [
               for (var weapon in widget.character.meleeWeapons) [
                 weapon.name,
-                weapon.length.toString(),
+                weapon.length.name,
```

Comment:
Powinieneś też zmienić niżej odpowiedni ColumnType na "name", zakładając że to jest taka normalna nazwa teraz